src/Provider/Facebook.php

```diff
@@ -103,7 +103,7 @@ public function getResourceOwnerDetailsUrl(AccessToken $token)
         $fields = [
             'id', 'name', 'first_name', 'last_name',
             'email', 'hometown', 'picture.type(large){url,is_silhouette}',
-            'cover{source}', 'gender', 'locale', 'link', 'timezone', 'age_range'
+            'gender', 'locale', 'link', 'timezone', 'age_range'
```

Comment:
```suggestion
            'gender', 'link', 'age_range'
```
Timezone and locale are also deprecated. 

https://developers.facebook.com/blog/post/2018/05/01/facebook-login-updates-further-protect-privacy/